scripts/components/reportsDashboards/build.sh

```diff
@@ -72,6 +97,12 @@ echo "BUILD MODULES FOR $PLUGIN_NAME"
 (cd ../../OpenSearch-Dashboards && yarn osd bootstrap)
 echo "BUILD RELEASE ZIP FOR $PLUGIN_NAME"
 (cd ../../OpenSearch-Dashboards/plugins/$PLUGIN_FOLDER && yarn plugin_helpers build)
+echo "DOWNLOADING CHROMIUM FOR $PLUGIN_NAME"
```

Comment:
can we have a mention here why we have to do this. i know why but like in code comment.

Reply:
sure